benchpress/tests/test_json_parser.py

```diff
@@ -71,6 +71,21 @@ def test_parse_empty_output(self):
         with self.assertRaises(ValueError):
             self.parser.parse(['garbage'], output, 1)
 
+    def test_parse_with_surrounding_garbage(self):
+        output = [
+            '',
+            '{ json-like looking part } }'
+            'some garbage here',
+            '{"hello": "world", "x": 42}',
+            'some other unreachable garbage',
```

Comment:
Can you extend this test case with another valid json object after the first one so that the existing assert will show that it only picks the first one

Reply:
good call, I'll add it.